README.md

```diff
@@ -1,2 +1,90 @@
-# coverter-go
-Convert AsyncAPI documents from older to newer versions.
+# AsyncAPI Converter
+
+Convert AsyncAPI documents from version 1.x to 2.0.0-rc1.
```

Comment:
Add more details regarding which file types are converted into which (default and --toYAML).